installer-guide/mac-install.md

```diff
@@ -4,38 +4,53 @@
 * Supported version: 0.5.7
 
 While you don't need a fresh install of macOS to use OpenCore, some users prefer having a fresh slate with their boot manager upgrades.
+This guide works for a release of you choice as long as it is listed in the catalog
 
 **Note for legacy users**
 
-* If you want to use OpenCore on a system without UEFI, please follow the [Legacy Install](https://dortania.github.io/OpenCore-Desktop-Guide/extras/legacy.html) section first, after you can continue following the **Base folder structure** section
+* To use OpenCore on an older system with no UEFI (Plain Bios):
+1. First follow the [Legacy Install](https://dortania.github.io/OpenCore-Desktop-Guide/extras/legacy.html) section
```

Comment:
Don't link to the site itself, please link to the file instead:
`[Legacy Install](/extras/legacy.md)`
This prevents the site from breaking if domains ever change